### ISSUE TITLE
Giving form covering content

### ISSUE THREAD

**Issue by mwllr** [ATLAS-Illinois/Illinois-Framework-Theme#939]
Accessibility violation since we are covering up the content.

The floating “giving” box will not let me access the bottom links on the page in the footer or sub-footer.
ACE website - on giving page only, the form covers the legal links:
![Screenshot 2023-09-12 at 10 51 33 AM](https://github.com/web-illinois/illinois_framework_theme/assets/54747669/6ce2557c-c703-4ce0-ba91-8ed4fe4fcb35)

home page without links covered:
![Screenshot 2023-09-12 at 10 51 25 AM](https://github.com/web-illinois/illinois_framework_theme/assets/54747669/9cdf91ee-36cd-4113-a25f-2047ebecd894)

**Comment by mwllr** [comment]
Student Affairs moved the total into the page content area before footer:
![Screenshot 2023-09-12 at 10 53 44 AM](https://github.com/web-illinois/illinois_framework_theme/assets/54747669/03f6cf0a-99ba-469d-8c14-9bd44a371483)

@Detzner  - would you be wiling to help with this on the framework level? @trubach

**Comment by Detzner** [comment]
Hi Melissa—

I did this at the request of the main stakeholder for the site (it’s a simple CSS style change). It’s been a while, but if I remember correctly her concern was that when people scroll all the way down, the total bar looked like part of the footer, and less connected to the fund descriptions.

Dick

From: Melissa Waller ***@***.***>
Reply-To: web-illinois/illinois_framework_theme ***@***.***>
Date: Tuesday, September 12, 2023 at 10:54 AM
To: web-illinois/illinois_framework_theme ***@***.***>
Cc: "Detzner, Richard" ***@***.***>, Mention ***@***.***>
Subject: Re: [web-illinois/illinois_framework_theme] Giving form covering content (Issue #939)


Student Affairs moved the total into the page content area before footer:
[Image removed by sender. Screenshot 2023-09-12 at 10 53 44 AM]<https://urldefense.com/v3/__https:/user-images.githubusercontent.com/54747669/267389444-03f6cf0a-99ba-469d-8c14-9bd44a371483.png__;!!DZ3fjg!7y_9h1MDb3AZEt9UecLYdFEwWj_0sefSLuHBtPgStqcYnRjnX2_7bsBUvjSyWbK-0VRmR6cUKySQBL5xF1wlB9CeOPE$>

@Detzner<https://urldefense.com/v3/__https:/github.com/Detzner__;!!DZ3fjg!7y_9h1MDb3AZEt9UecLYdFEwWj_0sefSLuHBtPgStqcYnRjnX2_7bsBUvjSyWbK-0VRmR6cUKySQBL5xF1wlD9PDxKE$> - would you be wiling to help with this on the framework level? @trubach<https://urldefense.com/v3/__https:/github.com/trubach__;!!DZ3fjg!7y_9h1MDb3AZEt9UecLYdFEwWj_0sefSLuHBtPgStqcYnRjnX2_7bsBUvjSyWbK-0VRmR6cUKySQBL5xF1wlRM5K6so$>

**Comment by trubach** [comment]
Closing - The giving form content type "Fund Listing" is deprecated and no longer needed since the UIF move to GiveCampus
